analyses/focal-cn-file-preparation/01-prepare-cn-file.R

```diff
@@ -76,24 +76,34 @@ process_annotate_overlaps <- function(cnv_df,
   cnv_gr <- cnv_df %>%
     GenomicRanges::makeGRangesFromDataFrame(keep.extra.columns = TRUE,
                                             starts.in.df.are.0based = FALSE)
-
-  # Create a data.frame with the overlaps between the CNV file and hg38 genome
-  # annotations
-  overlaps <- IRanges::mergeByOverlaps(cnv_gr, tx_exons)
-
-  # Create a data.frame with selected columns from the `autosome_overlaps`
+  
+  # Find the overlaps between the CNV file and hg38 genome annotations
```

Comment:
may want to make this note more specific to reference the GENCODE v27 filtered file or etc.